lib/DataObject/GridColumnConfig/Operator/AnyGetter.php

```diff
@@ -199,7 +199,7 @@ public function getParam1()
      */
     public function setParam1($param1)
     {
-        $this->param1 = SecurityHelper::convertHtmlSpecialChars($param1);
```

Comment:
Do we still need `SecurityHelper::convertHtmlSpecialChars`?

Reply:
Do you mean the Helper method itself?
Yes, I would say so. It's used in several controllers and helpers ...  I just removed it from models.

Reply:
we should also remove it from [Link Editable](https://github.com/pimcore/pimcore/blob/b159e547830b19f41d16d29c816c56a0a0facc29/models/Document/Editable/Link.php#L383)..right? and then i don't see any other usages of `convertHtmlSpecialChars` in the controllers and helpers. please double check this 😊

Reply:
I think we messed something up here 🤣 let me clarify:

The link editable uses the `sanitizeHtmlAttributes` method. This is only used by the editable and could be removed if we remove it from the link editable.

The other method is the `convertHtmlSpecialChars`. This method is used in some other placed. (eg. `DataObjectHelperController` or `ClassificationstoreController`)

Reply:
Ohh sorry, i was referring to the `sanitizeHtmlAttributes` later 😆. I think we should remove the sanitization from this model as well and kick the method from helper.

Reply:
If we remove the sanitizer here, we also have to rethink the frontend part. Afaik we only print out the saved string - that would open the door for HTML injects - right? 

Or did I miss something?

Reply:
In link editable we still need it, because there it's used for the frontend part ( `frontend()` method). There's no way to do it in frontend, so imho it's fine.

Reply:
yes, we still need to remove it from the getters.